diff --git a/mealie/core/config.py b/mealie/core/config.py
index 19a347d..fd5b3b0 100644
--- a/mealie/core/config.py
+++ b/mealie/core/config.py
@@ -19,16 +19,7 @@ DATA_DIR = os.getenv("DATA_DIR")
 
 
 def determine_data_dir() -> Path:
-    global PRODUCTION, TESTING, BASE_DIR, DATA_DIR
-
-    if TESTING:
-        return BASE_DIR.joinpath(DATA_DIR if DATA_DIR else "tests/.temp")
-
-    if PRODUCTION:
-        return Path(DATA_DIR if DATA_DIR else "/app/data")
-
-    return BASE_DIR.joinpath("dev", "data")
-
+    return Path("@StateDirectory@")
 
 @lru_cache
 def get_app_dirs() -> AppDirectories:
